src/OrleansCodeGenerator/GrainMethodInvokerGenerator.cs

```diff
@@ -265,7 +290,7 @@ private static MethodDeclarationSyntax GenerateInvokeMethod(Type grainType, Meth
             Type grainType,
             IdentifierNameSyntax grain,
             MethodInfo method,
-            IdentifierNameSyntax arguments)
+            ExpressionSyntax arguments)
```

Comment:
The change was introduced when I was (temporarily) using an expression (`method.Arguments`) instead of a variable (`arguments`). It is no longer needed, but I should have been using that type there anyhow - `IdentifierNameSyntax` is a sub-class of `ExpressionSyntax` and the code requires an expression, not an identifier.